.github/workflows/native-build-conan.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        warnings: [  "-Wall -Wextra -Werror " ]
+        warnings: [  "-Wall -Wextra"  ]
```

Comment:
Is it intentional that `-Werror` is dropped here?

Reply:
Yes, there are warnings in external dependencies that only happen in this build, and that otherwise would crash the build (previously those warnings were inactive).